packages/preact-query/src/__tests__/HydrationBoundary.test.tsx

```
@@ -3,7 +3,7 @@ import type { hydrate } from '@tanstack/query-core'
 import { queryKey, sleep } from '@tanstack/query-test-utils'
 import { render } from '@testing-library/preact'
 import { Suspense, startTransition } from 'preact/compat'
-import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+import { afterEach, beforeEach, describe, expect, vi, it } from 'vitest'
```

ESLint

[error] Member 'it' of the import declaration should be sorted alphabetically.

(sort-imports)